user-docs/documentation/docs/how-to-guides/3-migration-guides/2-babel-to-tsup.md

```diff
@@ -10,4 +10,61 @@ Coming soon...
 
 ## How?
 
-Coming soon...
+In order to migrate from Babel to Tsup:
+
+- Enable `import/no-extraneous-dependencies` in your eslint config:
+  ```js
+  {
+    files: ['**/src/**'],
+    excludedFiles: ['**/__tests__/**', '**/*.test.ts?(x)'],
+    rules: {
+      'import/no-extraneous-dependencies': [
+        'error',
+        {
+          devDependencies: false,
+          optionalDependencies: false,
+          peerDependencies: true,
+        },
+      ],
+    },
+  },
+  ```
```

Comment:
I think this is a bit different from simply migrating tsup

Here is a bit more contexte
```suggestion
Before migrating from Babel to Tsup, we strongly recommend adding eslint `no-extraneous-dependencies` rule.
This will ensure you don't forget to specify your dependencies in your `package.json`. If you forget any, tsup will bundle them resulting in bigger bundle size

- Enable `import/no-extraneous-dependencies` in your eslint config:
  ```js
  {
    files: ['**/src/**'],
    excludedFiles: ['**/__tests__/**', '**/*.test.ts?(x)'],
    rules: {
      'import/no-extraneous-dependencies': [
        'error',
        {
          devDependencies: false,
          optionalDependencies: false,
          peerDependencies: true,
        },
      ],
    },
  },
  ```
```

Reply:
This is clearer indeed